kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -20,7 +20,7 @@ define([
 
     function factory(config) {
         const viewOnly = config.viewOnly || false;
-        const { bus, workspaceId, paramIds, initialParams } = config;
+        const { bus, workspaceId, paramIds, initialParams, initialDisplay } = config;
```

Comment:
`initialDisplay` values should get passed to the different parameter instantiation components.